doc/processes/release_process.md

```diff
@@ -109,7 +109,9 @@ them.  They are merely points in time.
 Update the Extensions branches to pull the tagged levels from the `openj9` 
 & `openj9-omr` release branches.
 1. Rebuild the tagged levels and ensure they pass the quality bar defined by 
-AdoptOpenJDK.
+AdoptOpenJDK. These builds need to have `--with-milestone=fcs` (JDK8) or
```

Comment:
Please explain why these options are required and what they do.

Reply:
These configure options are what openjdk's makefiles use to set variables that determine what appears in the java -version output, as discussed a while back.. The default ones will mean you have "user" build info in the strings, username's etc...

Reply:
Thanks Andrew. To be clear, what I meant is that the document should be updated with that information.

Reply:
I've added a bit more info to address this - let me know if it's adequate or if more detail is required.